governance/program/src/state/realm_config.rs

```diff
@@ -0,0 +1,108 @@
+//! RealmConfig account
+
+use solana_program::{
+    account_info::AccountInfo, program_error::ProgramError, program_pack::IsInitialized,
+    pubkey::Pubkey,
+};
+
+use borsh::{BorshDeserialize, BorshSchema, BorshSerialize};
+
+use crate::{
+    error::GovernanceError,
+    state::enums::GovernanceAccountType,
+    tools::account::{get_account_data, AccountMaxSize},
+};
+
+/// RealmConfig account
+/// The account is an optional extension to RealmConfig stored on Realm account
+#[derive(Clone, Debug, PartialEq, BorshDeserialize, BorshSerialize, BorshSchema)]
+pub struct RealmConfigAccount {
+    /// Governance account type
+    pub account_type: GovernanceAccountType,
+
+    /// The realm the config belong to
+    pub realm: Pubkey,
+
+    /// Addin providing voter weights for community token
+    pub community_voter_weight: Option<Pubkey>,
```

Comment:
nit: can we call this `community_voter_weight_addin`?

Reply:
Done, much better this way. 
The filed name was a leftover from the time when the account was called RealmAddins and the suffix was redundant :)